ios/pod_helpers.rb

```diff
@@ -45,9 +45,9 @@ def find_file(file_name, current_dir)
 end
 
 def new_architecture_enabled?(options, react_native_version)
-  supports_new_architecture = supports_new_architecture?(react_native_version)
-  supports_new_architecture && ENV.fetch('RCT_NEW_ARCH_ENABLED',
-                                         options[:fabric_enabled] || options[:turbomodule_enabled])
+  return false unless supports_new_architecture?(react_native_version)
```

Comment:
RuboCop requires an empty line after guard clause:
```
Error: ios/pod_helpers.rb:48:3: C: [Correctable] Layout/EmptyLineAfterGuardClause: Add empty line after guard clause.
  return false unless supports_new_architecture?(react_native_version)
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
```suggestion
  return false unless supports_new_architecture?(react_native_version)

```